babel.config.ts

```diff
@@ -24,7 +25,7 @@ const config: TransformOptions = {
   plugins: [
     '@emotion/babel-plugin',
     '@babel/plugin-transform-runtime',
-    '@babel/plugin-transform-class-properties',
```

Comment:
Is it intentional to remove this?

Reply:
Whoops, forgot to put this one back in while I was experimenting. Not intentional